colmena/value_server/__init__.py

```diff
@@ -0,0 +1,26 @@
+"""Implementation of the value server"""
+
+from colmena.value_server.proxy import async_resolve_proxies
+from colmena.value_server.proxy import to_proxy
+from colmena.value_server.proxy import to_proxy_threshold
+from colmena.value_server.proxy import ObjectProxy
+
+from colmena.value_server.server import init_value_server
+from colmena.value_server.server import LRUCache
+from colmena.value_server.server import VALUE_SERVER_HOST_ENV_VAR
+from colmena.value_server.server import VALUE_SERVER_PORT_ENV_VAR
+
+global server
```

Comment:
Another thing to eventually change down the road: Allowing more multiple servers